src/fidesops/ops/schemas/manual_webhook_schemas.py

```diff
@@ -26,10 +26,16 @@ class Config:
         orm_mode = True
 
 
+if TYPE_CHECKING:
```

Comment:
didn't know about this built-in. Any particular reason to use this vs type ignore?

Reply:
good question, I was just running into mypy issues with Pydantic's `conlist`, and this still gives mypy something to work with without ignoring it altogether. 

https://mypy.readthedocs.io/en/stable/runtime_troubles.html#typing-type-checking
the `if TYPE_CHECKING` branch is only seen by mypy and the `else` branch is not seen by mypy and processed at runtime.

Reply:
oh cool, looks like it can be useful for those circular import issues we sometimes run into- https://mypy.readthedocs.io/en/stable/runtime_troubles.html#import-cycles . Although ideally we'd structure the code such that this isn't an issue...